src/app/lib/config/services/amharic.js

```diff
@@ -130,36 +130,28 @@ const service = {
   footer: {
     externalLink: {
       href: 'https://www.bbc.co.uk/help/web/links/',
-      text: 'Read about our approach to external linking.',
+      text: 'ስለ ውጪ ሊንኮች ያለን አቀራረብ፡፡',
     },
     links: [
       {
         href: 'https://www.bbc.com/news/help-41670342',
-        text: 'Why you can trust the BBC',
+        text: 'ቢቢሲን ለምን ማመን እንደሚገባዎ',
       },
       {
         href: 'https://www.bbc.com/terms',
-        text: 'Terms of Use',
-      },
-      {
-        href: 'https://www.bbc.co.uk/aboutthebbc/',
-        text: 'About the BBC',
+        text: 'የአጠቃቀም ደንብ',
       },
       {
         href: 'https://www.bbc.com/privacy/',
-        text: 'Privacy Policy',
+        text: 'የፕራይቬሲ ፖሊሲ',
       },
       {
         href: 'https://www.bbc.com/usingthebbc/cookies/',
-        text: 'Cookies',
-      },
-      {
-        href: 'https://www.bbc.com/accessibility/',
-        text: 'Accessibility Help',
+        text: 'ኩኪዎች',
       },
       {
         href: 'https://www.bbc.com/contact/',
-        text: 'Contact the BBC',
+        text: 'ቢቢሲን ያግኙ',
       },
     ],
     copyrightText: 'ቢቢሲ፡፡ ቢቢሲ ለሌሎች የኢንተርኔት ገጾች ኃላፊነት አይወስድም',
```

Comment:
This also needs updating as it doesn't match the translations on https://docs.google.com/spreadsheets/d/1IE6dvHfOJhYTIhC0kg8aRGnaj3fEg1NH9jQwo5NuUDQ/edit#gid=0